### DISCUSSION TITLE
How do I get the raw response about the plan?

### DISCUSSION THREAD

**Discussion by satoshinishimura-fc** [microsoft/semantic-kernel#2355] [Q&A]
I am using the SequentialPlanner to create the plan. Sometimes, I got the PlanningException: Create plan error: Unale to create plan. I think the SK planner tries to call non-existent function. I would like to see the raw response from GPT when creating the plan.
I tried to use `plan.ToPlanString()` but it privides very limited information.

**Comment by evchaki** [comment]
@satoshinishimura-fc  you can use telemetry to see what was returned - https://devblogs.microsoft.com/semantic-kernel/unlock-the-power-of-telemetry-in-semantic-kernel-sdk/
